.github/workflows/common-release.yml

```diff
@@ -83,3 +83,19 @@ jobs:
             export GIT_COMMITTER_DATE="$(git log -1 --format=%aD ${{ inputs.versionPrefix }}${SEMVER_VERSION})"
             git push origin "${{ inputs.versionPrefix }}${SEMVER_VERSION}" -f
           fi
+
+      - name: Create Issue on Failed workflow
+        if: ${{ failure() }}
+        run: |
+          if curl -sL --fail\
+              -X POST \
+              -H "Accept: application/vnd.github+json" \
+              -H "Authorization: Bearer ${{ secrets.GH_TOKEN }}" \
+              -H "X-GitHub-Api-Version: 2022-11-28" \
```

Comment:
Looks good but I think use the following syntax to specify the version https://github.com/terraform-ibm-modules/common-pipeline-assets/blob/main/.github/workflows/post-usage.yml#L33
What you have is probably better practice but I do not want to maintain the exact version. My approach just locks into the v3 API version so realistically they should bump that if there is a breaking API change. I am happy to hear if anyone else has thoughts on this.

Reply:
^^ fine with me, and I agree

Reply:
@daniel-butler-irl were you suggesting to remove the header `-H "X-GitHub-Api-Version: 2022-11-28"` ?